### DISCUSSION TITLE
I Did not receive purchase after License.

### DISCUSSION THREAD

**Discussion by harutobi** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#823] [Q&A]
I purchased Aldente in paddle
I did not license my e-mail
please help me. 
I sent to email (assist@paddle.com)
please check and answer please..
I hope Aldente...

```
purchase paid date: 2022-11-12 13:35:44
```

**Comment by MatthiasKerbl** [comment]
Hi @harutobi,

I am sorry you did not receive the purchasing emails.
If you already sent an email to Paddle support, they will take care of it and reply to you as soon as possible.

Just to clarify, there are multiple reasons why you did not get your license code after the purchase:
1. Sometimes it takes up to 30 Minutes for the purchase emails to go through.
2. The purchase emails end up in the junk email folder.
3. You had a typo in the email address during your purchase.
4. Failures of the payment networks/providers.

Cheers,
Matthias

**Reply by harutobi** [reply]
@MatthiasKerbl 
Thanks for the reply. :)
First, I will contact the reseller paddle customer center and ask for confirmation.
If it doesn't work out for you, please help me again.
I'll leave a comment here when the result comes out.
Thank you.

**Reply by harutobi** [reply]
@MatthiasKerbl 
It took a long time to check the payment information, but I solved it well.

The final result was that the email address was invalid and the license was not received.

I will use a good product well.

Thank you.

Have a nice day.

**Reply by MatthiasKerbl** [reply]
Hi @harutobi,

I am glad to hear that. :)

Cheers,
Matthias
